scss/_patterns_grid.scss

```
@@ -232,18 +243,45 @@
   .row--25-25-50 {
     @extend %vf-row;
 
-    .#{$grid-column-prefix} {
+    > .col {
       @include vf-grid-column($grid-columns-small);
 
       @media (min-width: $threshold-4-6-col) {
-        &:nth-child(-n + 2) {
-          @include vf-grid-column(calc($grid-columns-medium / 2));
+        &:nth-of-type(-n + 2) {
```

Why changing to `nth-of-type`? This would make it match nth `div` or whatever else element type is used. This does not mean nth element with `.col` class name unfortunately.

So I think `nth-child` is actually safer there, as it's closer to what CSS grid does by default (each child is put into one grid area).

---

So I'd suggest reverting back to `nth-child` in all occurences of this, unless there is some issue with nth-child in this case?

```suggestion
        &:nth-child(-n + 2) {
```

---

But `nth-child` would break if even a `<hr>` was present within the row. I think, even if not perfect this is actually more robust!

---

But you should not put anything that is not a column inside the row I think. Direct child of row should always be a column, and only in column should be anything else.

Even if technically it works right now, it may be unsafe to do it, as row expects column children, not anything else. And if we change the implementation of row at some point, having other children may break.

Anyway, doesn't matter to much, both work the same when markup is as expected, so I guess the question is what should happen if it's not as expected. And indeed `nth-of-type` is more flexible in this regard.

Let's leave it like that.